<launch>
  <node name="visit_my_home" pkg="tbm4_visit_my_home" type="visit_my_home_controller.py" output="screen" />
  <include file="$(find hearts_navigation)/launch/hearts_navigation_navigate.launch">
    <arg name="map_name" value="map_pal3" />
  </include>
  <param name="locations_json" type="string" value="$(find hearts_navigation)/data/locations.json" />
  <param name="output_path" value="$(find tbm4_visit_my_home)/output/" />
  <node pkg="roah_rsbb_comm_ros" type="comm" name="roah_rsbb_comm" respawn="true">
    <param name="team_name" type="string" value="Hearts" />
    <param name="robot_name" type="string" value="Tiago" />
    <param name="rsbb_key" type="string" value="FJ6LazOM" />
    <param name="rsbb_host" type="string" value="192.168.1.129" />
  </node>
  <include file="$(find hearts_person_tracker)/launch/hearts_person_tracker.launch">
  </include>
</launch>
